README.MD

```diff
@@ -1,97 +1,60 @@
-# Web-Components implementation for Wholesale
+# Zalando Wholesale Styleguide Components
+[![CircleCI](https://circleci.com/gh/wholesale-design-system/components/tree/release.svg?style=svg)](https://circleci.com/gh/wholesale-design-system/components/tree/release)
+[![Styleguide](https://img.shields.io/github/issues/wholesale-design-system/components.svg)](https://github.com/wholesale-design-system/components/issues)
+[![GitHub contributors](https://img.shields.io/github/contributors/wholesale-design-system/components.svg)]()
 
-## Aim
+## Motivation
 
-We in wholesale want the developers to choose frameworks they feel comfortable with. Still we want the UI to look and behave consistent. For that reason we decided to have components that can be used framework agnostic, web-components.
+At Zalando Wholesale we are developing a lot of web applications from small to mid-complex. As we are developing them in different cross-functional teams we wanted to guarantee the same look and feel in all of our applications independent from people and their development background. For guaranteeing same UI across applications and development teams we are developing a component-based Styleguide which provides a SCSS library and (P)React based components which can be all used independently. 
 
-## Usage
-`npm install` and then run
-`gulp build` to build the components.
-`npm run start` to run the demo page with react.
-`npm run start:preact` demo page with preact.
-`gulp test` for test / watcher
+In this repository we are developing the components which can be used later in your application. 
 
-## Demo
+## How it works
 
-In 'demo/index.html' you will find already build web-components. And see how you can integrate them in a vanilla setup.
+Basically the components in this repository are one part of the styleguide. We are not providing or installing any framework (react or preact) or styles [styles](https://github.com/wholesale-design-system/scss), therefore if you want to use the components you have to provide the framework and styles in your project environment.
 
-## Integration with Angular 2
+We have abstracted our imports from specific frameworks as we think developers should have the opportunity to decide which framework they want to use. 
 
-Check out the implementation of the header for an ng2 project: `https://github.bus.zalan.do/alpha/organizational-structure-frontend/blob/master/frontend/src/app.ts`.
+## Getting started (How to integrate)
 
-## How to extend
+The package is not yet published on npm as we are still working in a beta phase. 
 
-Create a folder under `webcomponents` with the name of your web-component. In there you can have .html, .scss and .js files that will be merged into one file by the build process.
+First of all you should install react / preact (if not already done) and our scss rules:
 
-### How do Web-Components work
+- `npm i -D -E preact prop-types` or `npm i -D -E react react-dom prop-types`
```

Comment:
Maybe a stupid question but why should someone load an extra library which behaves the same like reakt if react is already loaded?^^ So I gues no one will have an React app and plan to just use the components with preact^^